### PR TITLE
Upgrade to ember-cli 2.8

### PR DESCRIPTION
**Pull request by thoov**

### CODE REVIEW

File: config/ember-try.js [hunk 1]
```diff
@@ -1,6 +1,55 @@
 /*jshint node:true*/
 module.exports = {
-  // see package.json
-  // TLDR: 1.13.x ++
-  useVersionCompatibility: true
```
Comment (blimmer) [inline config/ember-try.js]: We should retain the 1.13.x + [version compatibility](https://github.com/yapplabs/ember-buffered-proxy/pull/28/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L57) as it gives us more test coverage than this change.

File: .travis.yml [hunk 2]
```diff
@@ -1,23 +1,38 @@
...
 install:
-  - npm install -g bower
   - npm install
   - bower install
 
 script:
-  - ember try:each
+  # Usually, it's ok to finish the test scenario without reverting
+  #  to the addon's original dependency state, skipping "cleanup".
+  - ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
```
Comment (blimmer) [inline .travis.yml]: This should still use the empty try version compatibility functionality merged with the .travis.yml upgrades made with 2.8.

File: package.json [hunk 3]
```diff
@@ -1,48 +1,42 @@
...
   "engines": {
     "node": ">= 0.10.0"
   },
-  "author": "Kris Selden, Stefan Penner & Yapp Labs",
-  "contributors": [
-    {
-      "name": "Ben Limmer",
-      "email": "hello@benlimmer.com"
-    }
-  ],
```
Comment (blimmer) [inline package.json]: Please retain the existing package.json configurations merged with dependency version updates.

**Comment by blimmer** [comment]
Also, thank you for helping out with this PR!

File: package.json (hunk 3 above)
Comment (thoov) [inline package.json]: Forgot about package.json. Added back the needed fields

File: package.json [hunk 4]
```diff
@@ -9,13 +9,13 @@
...
     "start": "ember server",
-    "test": "ember try:testall"
+    "test": "ember try:each"
   },
   "repository": "https://github.com/yapplabs/ember-buffered-proxy",
   "engines": {
     "node": ">= 0.10.0"
   },
-  "author": "Kris Selden, Stefan Penner & Yapp Labs",
+  "author": "Kris Selden, Stefan Penner & Yapp Labs",		 +  "author": "",
```
Comment (blimmer) [inline package.json]: ?

Comment (thoov) [inline package.json]: Ya just fixed that

**Comment by blimmer** [comment]
This still needs version compatibility restored

**Comment by thoov** [comment]
@blimmer Can you relook at this

**Comment by blimmer** [comment]
I fixed one last thing and will merge after tests pass - thanks!

**Pull request merged**